jobs/dtd-metrics/src/index.ts

```diff
@@ -50,16 +51,13 @@ async function main(): Promise<void> {
     publishedEServicesMetric,
     macroCategoriesPublishedEServicesMetric,
     top10MostSubscribedEServicesMetric,
-    top10MostSubscribedEServicesPerMacroCategoryMetric,
     top10ProviderWithMostSubscriberMetric,
   })
 
-  await bucket.uploadData(output, env.FILENAME)
-
+  bucket
+  output
+  // await bucket.uploadData(output, env.FILENAME)
   log('Done!\n')
-
-  await readModel.close()
```

Comment:
closing the connection is no more required?

Reply:
It is still required, my bad